### ISSUE TITLE
Editor crash when changing to big zoom in Script editor

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#23877]
**Godot version:**
3.1 f2cc969

**OS/device including version:**
Windows 10

**Issue description:**
When I zoom script editor to max(685%) and I try to zoom out, then editor crash without backtrace with this error
```
ERROR: get: FATAL: Index p_index=2 out of size (size()=2)
   At: ./core/cowdata.h:149
````

**Steps to reproduce:**
1. Zoom script editor to 400%-700%
2. Try to zoom to 100%

**Comment by akien-mga** [comment]
I could reproduce it only once (the first time) so far, console spammed with:
```
[...]
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
TOTAL BYTES: 1048560
NULL count: 1
CALL _update_callback: 1698
CALL _update_minimum_size: 1
CALL _sort_children: 261
CALL _on_theme_changed: 4
CALL _update_scrollbar_position: 1
CALL _tree_thumbnail_done: 8344
ERROR: push_call: Message queue out of memory. Try increasing 'message_queue_size_kb' in project settings.
   At: core/message_queue.cpp:56.
Failed method: FileSystemDock:_tree_thumbnail_done target ID: 7452
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
Object was deleted while awaiting a callback
[...]
```

**Comment by qarmin** [comment]
I tried to reproduce this in Ubuntu 18.10 but I was unable to do this.
It looks to be only Windows issue

**Comment by qarmin** [comment]
It works in 0cff752, probably fixed by #24447
